### PR TITLE
Restore linking of PureCN analysis scripts.

### PR DESCRIPTION
**Pull request by roryk**
The automatic updates removed the linking of the scripts. The dependencies were
pinned too hard for this formula and needed to be build on R 3.6 depite the
PureCN recipe not needing versions that were that new, so I removed the pinning
on those formulas.

I also added in checks to make sure the scripts are properly linked and
available.

Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/bioconductor-purecn/meta.yaml [hunk 1]
```diff
@@ -20,19 +20,19 @@ build:
 # Suggests: PSCBS, copynumber, BiocStyle, BiocParallel, knitr, optparse, TxDb.Hsapiens.UCSC.hg19.knownGene, org.Hs.eg.db, covr, testthat
 requirements:
   host:
-    - 'bioconductor-biocgenerics >=0.30.0,<0.31.0'
-    - 'bioconductor-biostrings >=2.52.0,<2.53.0'
+    - bioconductor-biocgenerics
+    - bioconductor-biostrings
```
Comment (dpryan79) [inline recipes/bioconductor-purecn/meta.yaml]: The pinnings were required by Bioconductor for proper compatibility, they can't be removed. I think BiocGenerics requires R 3.6, which will be available in conda in July.

**Comment by roryk** [comment]
Thanks-- I figured the pinning was there for a reason. Can we not temporarily unpin them and then repin them when 3.6 is all set? Or are we stuck until July?

**Comment by dpryan79** [comment]
We’re stuck until July.

**Comment by roryk** [comment]
Ok, thanks!

**Pull request closed without merging**